cypress/integration/unit_tests/click_feature_test.js

```diff
@@ -1,90 +1,201 @@
+   */
+  function getDataFeatures() {
+    return cy.wrap(null).then(() => {
+      const features = [];
+      map.data.forEach((feature) => features.push(feature));
+      return features;
+    });
+  }
 
-    mockTable.verify();
-  });
+  /**
+   * Clicks on the map to pop up feature1 and asserts presence.
+   * @return {Cypress.Chainable}
+   */
+  function clickAndVerifyBlockGroup() {
+    cy.get('#test-map-div').click(0, 0);
+    cy.get('#test-map-div').should('contain', 'SCORE: 1');
+    cy.get('#test-map-div').should('contain', 'my block group');
+    cy.get('.google-visualization-table-tr-sel')
+        .find('[class="google-visualization-table-td"]')
+        .should('have.text', 'my block group');
+    return assertFeatureShownOnMap(feature1Corners);
+  }
+
+  /**
+   * Asserts that there is exactly one feature on the map, and that it has the
+   * given geometry.
+   * @param {Array<number>} expectedFeatureCorners The expected coordinates of
+   *     the feature, in the same form as {@link createFeatureFromCorners}
+   * @return {Cypress.Chainable}
+   */
+  function assertFeatureShownOnMap(expectedFeatureCorners) {
+    return getDataFeatures().then((features) => {
+      expect(features).to.have.length(1);
+      const feature = features[0];
+      expect(feature.getGeometry()).to.not.be.null;
+      const rings = feature.getGeometry().getArray();
+      expect(rings).to.have.length(1);
+      // chai is bad at equality on google.maps.LatLng objects, so convert.
+      const coordinates = rings[0].getArray().map(
+          (latlng) => ({lng: latlng.lng(), lat: latlng.lat()}));
+      const expected = [
+        {
+          lng: expectedFeatureCorners[0],
+          lat: expectedFeatureCorners[1],
+        },
+        {
+          lng: expectedFeatureCorners[2],
+          lat: expectedFeatureCorners[1],
+        },
+        {
+          lng: expectedFeatureCorners[2],
+          lat: expectedFeatureCorners[3],
+        },
+        {
+          lng: expectedFeatureCorners[0],
+          lat: expectedFeatureCorners[3],
+        },
+      ];
+      expect(coordinates).to.eql(expected);
+    });
+  }
 });
+
+/**
+ * Creates a rectangular test feature, with properties filled in from the
+ * given coordinates.
+ * @param {number} west West-most (smallest longitude) coordinate of feature
+ * @param {number} south South-most (smallest latitude) coordinate of feature
+ * @param {number} east East-most (largest longitude) coordinate of feature
+ * @param {number} north North-most (largest latitude) coordinate of feature
+ * @return {ee.Feature}
+ */
+function createFeatureFromCorners(west, south, east, north) {
+  const polygonCoordinates =
+      [west, south, west, north, east, north, east, south];
+  let result = ee.Feature(ee.Geometry.Polygon(polygonCoordinates));
+  for (let i = 0; i < tableHeadings.length; i++) {
+    result = result.set(
+        tableHeadings[i], polygonCoordinates[i % polygonCoordinates.length]);
```

Comment:
Setting each column in the table to some function of the polygon coords?

Reply:
Yeah, just an arbitrary value that will vary based on the passed-in polygon.

Reply:
geoid needs to be unique, so used "west" coordinate. After that, seemed just as easy as not to initialize with remaining coordinates. Added a comment.